packages/react/src/index.ts

```diff
@@ -206,3 +206,6 @@ export type {PageHeaderProps} from './PageHeader'
 export {SSRProvider, useSSRSafeId} from './utils/ssr'
 export {default as sx, merge} from './sx'
 export type {SxProp} from './sx'
+
+export {FilteredActionList} from './FilteredActionList'
+export type {FilteredActionListProps} from './FilteredActionList'
```

Comment:
Question for reviewer: 

Should we export this from `experimental` instead? It isn't really experimental because it's been what powers SelectPanel from the start.

But if we want to treat this as a temporary solution, then it feels okay

Reply:
I'm down with experimental if it means we won't have to go through a deprecated flow 😂 But if we still have to go through that, then stable is fine 👍 

(Separate from this PR) this could be a good example for spots where an `unstable_*` convention could help out without having to couple things to entry points

Reply:
My only comment is that it would be great to treat it as an experimental/unstable component so that we don't have to worry about semver in case if we need to make any changes for the SelectPanel remediation.

Reply:
Valid!